package.json

```diff
@@ -33,7 +33,9 @@
     "start:storybook:dev": "NODE_ENV=development  start-storybook --port 80",
     "dev": "concurrently --raw \"yarn start:storybook:dev\" \"yarn watch:lint:scss\"",
     "test": "cypress run",
-    "cypress:open": "cypress open"
+    "cypress:open": "cypress open",
+    "codegen:dbc:gateway": "CODEGEN_SCHEMA_AGENCY=775100 CODEGEN_SCHEMA_PROFILE=opac graphql-codegen --config dbc-gateway.codegen.yml",
```

Comment:
Hardcode agency id?

Reply:
It is used for constructing the schema url form where we generate the code and types from.
This is the info that we got so far, from where to retrieve the schema.
So why not just use a hardcoded url, one might ask?
This is just going one step further and parameterising the agency id and profile.
And I put the the parameters directly into package.json in for helping the developers.
Otherwise you would need to export the parameters every time in your console session or implement usage of .env.
But since there is nothing secret here I choose to put it as a part of the command.

Reply:
But should we here than use 100200 (bibliotek.dk's agency id)?

Reply:
Yes. Makes sense. I'll change that